### PR TITLE
TOOLS-125 enable nightly pytest execution

### PR DESCRIPTION
**Pull request by brennerm**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -31,117 +36,157 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: wlan-testing
+        ref: staging-wifi-2047
```
Comment (jaspreetsachdev) [inline .github/workflows/nightly.yml]: Shouldn't this be master?

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -31,117 +36,157 @@ jobs:
...
+      run: ./sync_repos.bash
+
+    - name: create configuration.py
+      working-directory: wlan-testing
+      run: |
+        cat << EOF > configuration.py
+        ${{ secrets.LAB_CONFIGURATION }}
+        EOF
+
+    - name: create Dockerfile
```
Comment (jaspreetsachdev) [inline .github/workflows/nightly.yml:56]: We have a dockerfile now in docker directory of this repository. Is it possible to use that?

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -31,117 +36,157 @@ jobs:
...
+
     # build and push docker image
     - name: docker login
-      shell: bash
       run: docker login ${{ env.DOCKER_SERVER }} -u ${{ env.DOCKER_USER_NAME }} -p ${{ env.DOCKER_USER_PASSWORD }}
     - name: build docker image
-      shell: bash
-      run: docker build -t ${{ env.DOCKER_SERVER }}/cloud-sdk-nightly:pytest-${{ github.run_id }} -f wlan-testing/pytest/Dockerfile .
+      working-directory: wlan-testing
+      run: docker build -t ${{ env.DOCKER_SERVER }}/cloud-sdk-nightly:pytest-${{ github.run_number }} .
```
Comment (jaspreetsachdev) [inline .github/workflows/nightly.yml:81]: So this means that we can delete the docker file from the repo and have our README.md file make use of generated docker images for the testing repo. Thoughts?

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (brennerm) [reply .github/workflows/nightly.yml]: @jaspreetsachdev Think you have been looking at an old state of this file.

File: .github/workflows/nightly.yml (hunk 3 above)
Comment (brennerm) [reply .github/workflows/nightly.yml]: I would keep the "how to build" section and a one that explains how to pull the latest prebuilt image.

**Review by jaspreetsachdev — DISMISSED** [review]

**Pull request merged**